dev/NavigationView/NavigationView.cpp

```diff
@@ -1779,10 +1779,90 @@ void NavigationView::OnNavigationViewItemKeyDown(const winrt::IInspectable& send
     }
 }
 
+void NavigationView::ArrowKeyNavigationPolyfill(const winrt::NavigationViewItem& nvi, const winrt::KeyRoutedEventArgs& args)
+{
+    switch (args.Key())
+    {
+    case winrt::VirtualKey::Down:
```

Comment:
do we need to handle left/right for top nav ?

Reply:
We don't need to because the arrow key navigation issue was with it scrolling instead of navigating to the next item. This doesn't come up in TopNav left/right keys because it can never scroll.

---
In reply to: [367155739](https://github.com/microsoft/microsoft-ui-xaml/pull/1850#discussion_r367155739) [](ancestors = 367155739)